:simple-python: &nbsp;&nbsp;&nbsp; :simple-gnubash: &nbsp;&nbsp;&nbsp; :simple-r: &nbsp;&nbsp;&nbsp; :simple-git: &nbsp;&nbsp;&nbsp; :simple-github: &nbsp;&nbsp;&nbsp; :simple-docker: &nbsp;&nbsp;&nbsp; :fontawesome-brands-aws: &nbsp;&nbsp;&nbsp; :material-database: &nbsp;&nbsp;&nbsp; :simple-tensorflow: &nbsp;&nbsp;&nbsp; :simple-pytorch: &nbsp;&nbsp;&nbsp; :simple-scikitlearn: &nbsp;&nbsp;&nbsp; :simple-pandas: &nbsp;&nbsp;&nbsp; :simple-numpy: &nbsp;&nbsp;&nbsp; :simple-opencv: &nbsp;&nbsp;&nbsp; :simple-plotly: &nbsp;&nbsp;&nbsp; :simple-markdown: 



## <span style="color:purple">Machine Learning Skills</span > :robot:

<table>
  <thead>
    <tr>
      <th>Machine Learning</th>
      <th>Deep Learning</th>
      <th>Computer Vision</th>
    </tr>
  </thead>
<tbody>
    <tr>
      <td>
        Linear Regression, 
        Logistic Regression,
        Na√Øve Bayes, KNN,
        Support Vector Machine, 
        Decision tree,
        Random forest, Bagging,Boosting,
        Clustering, PCA, Recommender Systems
      </td>
      <td>
        Artificial Neural Networks (ANN), 
        Recurrent neural network (RNN),
        Transfer learning 
      </td>
      <td>
        Convoluated Neural Network (CNN),
        Object detection,
        Image segmentation
      </td>
    </tr>
</tbody>
<thead>
    <tr>
        <th>Natural Language processing</th>
        <th>ML Libraries</th>
        <th>Statistical Techniques</td>
     </tr>
</thead>
    <tbody>
        <tr>
            <td>
                TF-IDF, Word Embeddings, Long short-term memory network (LSTM),  Sentiment Analysis
            </td>
            <td>
                NumPy, Scikit-Learn, TensorFlow, Keras, OpenCV, NLTK
            </td>
            <td>
                Hypothesis Testing, Exploratory Data Analysis, Outlier Detection, Imputations, Feature Engineering
            </td>
        </tr>
    </tbody>
</table>

## <span style="color:purple">Computing Skills</span> :computer:
<table>
<thead>
    <tr>
      <th>Programming Languages</th>
      <th>Visualization Techniques</th>
      <th>Database Language</th>
    </tr>
</thead>
    <tbody>
        <tr>
            <td>Python, R/Bioconductor, Unix/Shell</td>
            <td>Seaborn, Pandas, Matplotlib, Plotly Dash </td>
            <td>SQL</td>
        </tr>
    </tbody> 
<thead>
    <tr>
        <th>Version control</th>
        <th>Cloud Computing</th>
        <th>Containerization</th>
</tr>
</thead>
<tbody>
    <tr>
        <td>Git, GitHub</td>
        <td>AWS EC2, ECR, S3, RDS, SageMaker, Lambda, Autoscaling, CloudWatch, SNS </td>
        <td>Docker</td>
    </tr>
</tbody>
</table>

## <span style="color:purple">Bioinformatics Skills</span> :dna:
<table width="100%">
<thead>
    <tr>
        <th>Next Generation Sequence Data Analysis</th>
    </tr>
</thead>
<tbody>
    <tr> 
        <td> <b>Quality Control</b>: FastQC, Samtools, Cutadapt, Bedtools, Trimommatic, PICARD </td>
    </tr>
    <tr> 
        <td> <b>De novo genome assembly</b>: Abyss, Platanus, SOAP denovo, MASURCA, SPAdes, metaSPADES, Trinity, GapClosure </td>
    </tr>
    <tr> 
        <td><b>Variant calling</b>: GATK, VarScan, mpileup </td>
    </tr>
    <tr> 
        <td><b>Single-cell analysis</b>: Cell Ranger, Seurat </td>
    </tr>
    <tr> 
        <td><b>Methylation data analysis</b>: HOMER, MAC, BISMARK </td>
    </tr>
    <tr> 
        <td><b>Metagenomics data analysis</b>: QIIME, MG-RAST, UCLUST, UPARSE, VSEARCH, METAGENassitsit, MEGAN </td>
    </tr>
    <tr> 
        <td><b>Plasmid detection</b>: Recycler, SPADES, plasmidSPADES</td>
    </tr>
    <tr> 
        <td><b>Annotation Tools</b>: Prodigal, Augustus, RepeatMasker, GO , KASS, KEGG, COG Pathway analysis, SSR, Pfam </td>
    </tr>
    <tr> 
        <td><b>Visualization Tools</b>: IGV genome browser, Circos, FigTree, MEME, GenomeVx, Brig, CONTIGuator </td>
    </tr>
</tbody>
</table>
